### PR TITLE
Tx/hd 125

### PR DESCRIPTION
**Pull request by lawauditswe**
Tidied up homepage.

### CODE REVIEW

File: src/containers/DemoContainer.tsx [hunk 1]
```diff
@@ -30,6 +31,24 @@ const DemoContainer: React.FC = () => {
   // conditional rendering of text when "Clear Cache" button is clicked
   const [cacheIsClear, setCacheIsClear] = React.useState<Boolean>(false);
 
+  // conditional rendering of DenoStore demonstration paragraph when info icon is clicked
+  const [demoParagraph, setDemoParagraph] = React.useState<Boolean>(false);
+
+  // state of the button based on whether Demo Explanation is visible or hidden
+  const [demoExplanationClicked, setDemoExplanationClicked] =
```
Comment (jakedoublev) [inline src/containers/DemoContainer.tsx]: Can we use just one piece of state here? demoExplanationOpen as demoExplanationClicked

**Review by jakedoublev — CHANGES_REQUESTED** [review]
one comment/review needed

File: src/containers/DemoContainer.tsx (hunk 1 above)
Comment (lawauditswe) [reply src/containers/DemoContainer.tsx]: Resolved.

**Review by jakedoublev — APPROVED** [review]
Good work TX. Looks great.

**Pull request merged**